apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -2953,4 +2956,39 @@ update-ledger-entry-panel .ledger-main .mat-form-field {
         }
     }
 }
+.daybook-advance-search {
+    .mat-form-field-appearance-fill {
+        .mat-form-field-flex {
+            padding: 1px 0px 0px 0px !important;
+        }
+        .mat-form-field-infix {
+            padding: 0;
+            border-top: 0px;
+
+            input.mat-input-element {
+                padding-left: 10px !important;
+                width: 96% !important;
+            }
+        }
+    }
+    giddh-datepicker {
+        .mat-form-field {
+            width: 100%;
+        }
+    }
+    .mat-form-field {
+        .mat-form-field-wrapper {
+            padding-bottom: 0px;
+        }
+    }
+}
+.export-daybook {
+    .mat-raised-button {
+        padding-left: 10px !important;
```

Comment:
@ravinderwalkover we have classes pd-l1 and pd-r1 for this. please use that @ravinderwalkover

Reply:
Done